docker/Dockerfile.fedora

```diff
@@ -0,0 +1,23 @@
+ARG  from
+FROM ${from}
+
+ENV  CFLAGS=-w CXXFLAGS=-w
+
+RUN dnf install -y \
+    gzip \
+    diffutils \
+    fontconfig-devel \
+    freetype-devel \
+    gcc \
+    gcc-c++ \
+    libX11-devel \
+    libXext-devel \
+    libXrender-devel \
+    libjpeg-devel \
+    libpng-devel \
+    make \
+    openssl-devel \
+    perl \
+    zlib-devel \
+    qtwebkit-devel \
```

Comment:
Why do you need this? If not for this package, you could use `Dockerfile.centos` as-is (_I think yum is still supported on Fedora, right?_).

If this pulls in packages which are required, I'd want them to be explicitly listed instead of `qtwebkit-devel`.

Reply:
> Why do you need this? If not for this package, you could use `Dockerfile.centos` as-is (_I think yum is still supported on Fedora, right?_).

In fact, it's not needed. I thought it solved a problem when compiling qt but that's not the case.

Regarding using yum in Fedora, yes, you can use it. Currently /usr/bin/yum is a symbolic link to dnf-3, same as /usr/bin/dnf

> If this pulls in packages which are required, I'd want them to be explicitly listed instead of `qtwebkit-devel`.

`qtwebkit-devel` requires `qt-devel` but it is not needed anyway.